template/scripts/build-app.sh

```diff
@@ -0,0 +1,35 @@
+#!bin/bash
+
+# Script will be showed below:
+
+# Reset
+Color_Off='\033[0m'       # Text Reset
+
+# Regular Colors
+Red='\033[0;31m'          # Red
+Green='\033[0;32m'        # Green
+
+TYPE=$1
+
+function buildApp() {
+  cd ios && fastlane build type:build env:$1 && cd .. && cd android && fastlane build type:build env:$1
+}
+
+function main() {
+
+  if [ -z $TYPE ]; then
+    read -p 'Enter your environment: ' env
```

Comment:
I suggest that we should list all environments to make it cleaner, like this: Enter your environment (dev, staging, production)

Reply:
You mean cleaner or more clearer ?

Reply:
I mean clearer. I'm sorry if I confused you about that

Reply:
Updated